tests/framework/PostProcessors/LimitSurface/tests

```diff
@@ -15,7 +15,7 @@
   output = 'limitSurface_integral/LimitSurfaceUnWeightedPb_dump.xml limitSurface_integral/LimitSurfaceWeightedPb_dump.xml'
   csv = 'limitSurface_integral/LimitSurfaceWeightedPb_dump.csv limitSurface_integral/LimitSurfaceUnWeightedPb_dump.csv'
   max_time = 300
-  rel_err = 0.0001
+  rel_err = 0.001
```

Comment:
why?

Reply:
In the input file ```tests/framework/PostProcessors/LimitSurface/test_LimitSurface_and_integral.xml``` , the Tolerance is 0.001 so we are checking to the tolerance, instead of ten times more accurate than the tolerance.